build.sbt

```diff
@@ -40,6 +40,7 @@ val commonSettings = Seq(
   sonatypeProfileName := "com.spotify",
   licenses := Seq("Apache 2" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt")),
   homepage := Some(url("https://github.com/spotify/noether")),
+  fork in run := true,
```

Comment:
Hey @richwhitjr curious about this 😛 is it needed?

Reply:
Seeing a serialization bug so this was my first offering to the serialization overlord.  I get this when running the tests

```java.lang.ClassCastException: cannot assign instance of scala.collection.immutable.List$SerializationProxy to field com.spotify.noether.MetricCurvePoints.points of type scala.collection.immutable.List in instance of com.spotify.noether.MetricCurvePoint``

My best guess is that is because we test serialization in the unit tests but not using Kryo?  Looks like standard java serialization.